code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -379,7 +379,27 @@
 	var/static/list/color_list = list("#FF0000","#FF8C00","#FFFF00","#00FF00","#00BFFF","#0000FF","#9400D3")
 	icon_state = "rainbow_laser"
 	item_state = null
+	slot_flags = SLOT_BELT
+	charge_cost = 100
```

Comment:
defines

Reply:
confused what you want with this one. slot_flags = slot_belt is like that through all the code

Reply:
I mean you can use defines for the cost

Reply:
But why?